playwright/pages/components/BasePage.ts

```diff
@@ -0,0 +1,39 @@
+import { Locator, Page } from "@playwright/test";
+
+import { Menu } from "./Menu";
+
+export class BasePage {
+  readonly page: Page;
+  readonly menu: Menu;
+  readonly body: Locator;
+  readonly breadcrumb: Locator;
+
+  constructor(page: Page) {
+    /**
+     * Components
+     */
+    this.page = page;
+    this.menu = new Menu(page);
+
+    /**
+     * Static locators
+     */
+    this.body = page.getByRole("main");
+    this.breadcrumb = page.getByLabel("Breadcrumb");
+  }
+
+  /**
```

Comment:
I know, I would like to discuss that, I really want to have something that will be consistent. Seeing Prismic codebase all around, and I'm afraid we will just write stuff all around and POM models start to be hard to read and maintain after :/

I'm trying here to prevent one problem of the POM model.